### PR TITLE
Add rtc functionality

### PR DESCRIPTION
**Pull request by Taylorr82**

### CODE REVIEW

File: equipment_handler/source/rtcmk.c [hunk 1]
```diff
@@ -303,3 +280,572 @@ int RTCMK_ReadHours(/*I2C_TypeDef *i2c,*/
...
+
+  ret = RTCMK_RegisterGet(addr,RTCMK_RegMonth,&tmp);
+  if (ret < 0)
+  {
+    return(ret);
+  }
+
+  tmp &= _RTCMK_MONTH_MONTH_MASK;
+
+  *val = ((tmp & 0xF0) >> 4) * 10 + (tmp & 0x0F);
```
Comment (arrooney) [inline equipment_handler/source/rtcmk.c]: What is this doing and could we make it a macro

**Review by arrooney — COMMENTED** [review]
Looks good - any ideas about how to test?

File: equipment_handler/source/rtcmk.c (hunk 1 above)
Comment (Taylorr82) [reply equipment_handler/source/rtcmk.c]: I don't know what it does so I didn't want to touch it

Comment (Taylorr82) [reply equipment_handler/source/rtcmk.c]: Testing can be accomplished with the ground station code. which I'm working on now

Comment (Taylorr82) [reply equipment_handler/source/rtcmk.c]: The alarm functionality may or may not be used. but it could be tested with the scheduler if we decide to use it

Comment (arrooney) [reply equipment_handler/source/rtcmk.c]: Where is it from then?

**Review by arrooney — APPROVED** [review]
Approving, and am keen to see some tests.

**Pull request merged**